js/logging.js

```diff
@@ -127,7 +128,8 @@ loggingModule.factory(
                         data: angular.toJson({
                             url: $window.location.href,
                             message: message,
-                            type: "debug"
+                            type: "debug",
+                            desc: desc
```

Comment:
Actually `null` would be too logical for JS. Javascript removes the key/value if the value is undefined.
```
x = function(dev) { return JSON.stringify({'a' : 'b', 'blah' : dev}); }
function (dev) { return JSON.stringify({'a' : 'b', 'blah' : dev}); }
x()
"{"a":"b"}"
```